index.html

```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
+                <nav class="flex-item flex-item1"><h4>Ruby on Rails</h4> </nav>
+                <nav class="flex-item flex-item2"><h4>CSS</h4></nav>
+                <nav class="flex-item flex-item3"><h4>JavaScrit </nav>
+                <nav class="flex-item flex-item4"><h4>HMTL</h4></nav>
+            </nav>
+
+            <nav class="grid-item last">
+                    <button class="button1" type="button">See project</button>
+            </nav>
+        </nav>
+        
+    </section>
+    <section class="about">
+            <nav class="grid-container-about">
+                <nav class="First-image">
+                    <img src="icon/illustration 1- about me-mobile.svg" class="upper_img">
+                 </nav> 
+                <h1 class="upper_word"><br>About me</h1>
+                <nav class="about_text"> 
+                    <p> I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                    </p>
+                </nav>
+                <nav class="grid-image">
+                        <nav class="second-image">  
+                            <img src="icon/illustration 2-about me mobile.svg">
+                        </nav>  
+                        <nav class="resume_button"> 
+                            <nav class="grid-item resume">
+                                <button class="button-resume" type="button">Get my resume</button>
+                            </nav>
+                        </nav>
+                </nav>
+            </nav>
+            <nav class="grid-container-ab">
+                <nav class="grid-language">
+                    <img src="icon/icon-languages.svg">
+                    
+                </nav>
+                <nav class="title"><h3>Languages</h3></nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-about1"><h4>JavaScript</h4> </nav>
+                    <nav class="flex-about flex-about2"><h4>Ruby</h4></nav>
+                    <nav class="flex-about flex-about3"><h4>HTML</h4></nav>
+                    <nav class="flex-about flex-about4"><h4>CSS</h4></nav>
+                </nav>
+            </nav>
+            <nav class="grid-container-ab">
+                <nav class="grid-language">
+                    <img src="icon/icon-frameworks.svg">
+                    
+                </nav>
+                <nav class="title"><h3>Frameworks</h3></nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutjs"><h4>React.js</h4> </nav>
+                    <nav class="flex-about flex-about2"><h4>Ruby on Rails</h4></nav>
+                    <nav class="flex-about flex-about3"><h4>RSpec</nav>
+                </nav>
```

Comment:
- [ ] This is an unordered list section. For list items, it is best to use `ul` and `li` tags. `nav` tag is specific for `navigation menu`.
- As Example:
`<ul>`
`  <li></li>`
`  <li></li>`
`</ul>`